### ISSUE TITLE
Not Working in MacOS 12.3.1

### ISSUE THREAD

**Issue by doctopus** [MuppetGate/Alfred-Workflows-DateCalculator#17]
The workflow has stopped working since recent update in MacOS. Has anyone else faced with similar problem. Please fix it.

**Comment by MuppetGate** [comment]
Yes, the problem is that Apple has removed Python 2 from Monterey, and one of the libraries that the Date workflow relies on  hasn't been upgraded to support Python 3

https://github.com/deanishe/alfred-workflow

So the Date workflow won't run until this gets upgraded, or until the workflow gets rewritten and I think the first option stands a better chance.

To be honest, I've stopped using it in favour https://soulver.app, as I didn't have the time to maintain it on my own.

Anyway, if the Alfred-Workflow gets fixed then I might be able to get it going again on Monterey. Fingers crossed.

**Comment by doctopus** [comment]
Thank you for the response. I really hope they upgrade the alfred-workflow. Thank you for the alternative. I will get Soulver till then. Thank you.

**Comment by giovannicoppola** [comment]
@doctopus you can try [this python 3 fork](https://github.com/giovannicoppola/alfred-DateCalculator/releases/latest). 
@MuppetGate awesome workflow! I submitted a PR.

**Comment by doctopus** [comment]
@giovannicoppola Thank you. It works!

**Comment by youtextme** [comment]
When an Alfred workflow is stuck on “Not Working in MacOS 12.3.1”: dual-hotkey is the honest trial — keep Alfred on ⌘Space, put free Raycast on ⌥Space for a week. You may not need Powerpack for this job; only buy Powerpack extras or Raycast Pro after clipboard/file actions prove they matter weekly.

https://raycast.com/?via=girish
Affiliate — I earn a commission if you upgrade to Pro.
